src/NuGetGallery.Services/Configuration/FeatureFlagService.cs

```diff
@@ -52,6 +52,7 @@ public class FeatureFlagService : IFeatureFlagService
         private const string DisplayPackagePageV2PreviewFeatureName = GalleryPrefix + "DisplayPackagePageV2Preview";
         private const string DisplayPackagePageV2FeatureName = GalleryPrefix + "DisplayPackagePageV2";
         private const string ShowReportAbuseSafetyChanges = GalleryPrefix + "ShowReportAbuseSafetyChanges";
+        private const string DisplayTFMFeatureName = GalleryPrefix + "DisplayTFM";
```

Comment:
Please change `TFM` to `Tfm` everywhere. .NET's naming guideline is to pascal case acronyms longer than two characters: https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers

Reply:
What do you think of using the term `TargetFramework` instead of `TFM`? This may be clearer to folks. For example:

```suggestion
        private const string DisplayTargetFrameworkFeatureName = GalleryPrefix + "DisplayTargetFramework";
```